src/index.ts

```diff
@@ -61,6 +67,8 @@ export interface IClientConfig {
    * Poll for new configurations even if the initial configuration request failed. (default: false)
    */
   pollAfterFailedInitialization?: boolean;
+
+  disablePolling?: boolean;
```

Comment:
Consider passing a `persistentStore` instead which implements `isExpired` the way you want it:

* interface: https://github.com/Eppo-exp/js-client-sdk/blob/main/src/index.ts#L79
* we have an implementation of a `chrome.storage` backed store that supports a "cooldown" for example: https://github.com/Eppo-exp/js-client-sdk/blob/main/src/chrome.configuration-store.ts#L14
* users on aws lambda can provide a store that reads from dynamodb or redis to avoid fetching from CDN